lib/ts/recipe/session/types.ts

```diff
@@ -160,7 +160,9 @@ export interface ErrorHandlerMiddleware {
 }
 
 export interface TokenTheftErrorHandlerMiddleware {
-    (sessionHandle: string, userId: string, request: BaseRequest, response: BaseResponse): Promise<void>;
+    (sessionHandle: string, userId: string, tenantId: string, request: BaseRequest, response: BaseResponse): Promise<
```

Comment:
don't need to pass in tenantId here cause you can get it from the sessionHandle (check that the core doesn't revoke the session on this error).

Probably also add a test case somewhere that checks that you can get the tenantId from the sessionHandle in this callback.

Reply:
Add missing test case.

Reply:
Done adding test case